data/en/test_section_summary.json

```diff
@@ -0,0 +1,282 @@
+                                "id": "first-name",
+                                "label": "First name",
+                                "mandatory": true,
+                                "type": "TextField",
+                                "validation": {
+                                    "messages": {
+                                        "MANDATORY_TEXTFIELD": "Please enter a name or remove the person to continue"
+                                    }
+                                }
+                            },
+                            {
+                                "alias": "last_name",
+                                "id": "last-name",
+                                "label": "Last name",
+                                "mandatory": false,
+                                "type": "TextField"
+                            }
+                        ]
+                    }]
+                }]
+            }]
+        },
+        {
+            "id": "household-full-names-section",
+            "title_from_answers": [
+                "first-name",
+                "last-name"
+            ],
+            "groups": [{
+                "blocks": [{
+                        "type": "Question",
+                        "id": "repeating-block-1",
+                        "title": "{{answers.first_name[group_instance]}} {{answers.last_name[group_instance]}}",
+                        "questions": [{
+                            "id": "over-16-question",
+                            "title": "Is the person above over 16?",
+                            "description": "",
+                            "type": "General",
+                            "answers": [{
+                                "q_code": "5",
+                                "id": "what-is-your-age",
+                                "type": "Radio",
+                                "label": "",
+                                "mandatory": false,
+                                "options": [{
+                                        "label": "Yes",
+                                        "value": "Yes"
+                                    },
+                                    {
+                                        "label": "No",
+                                        "value": "No"
+                                    }
+                                ]
+                            }]
+                        }]
+                    },
+                    {
+                        "type": "Question",
+                        "id": "repeating-block-2",
+                        "title": "{{answers.first_name[group_instance]}} {{answers.last_name[group_instance]}}",
+                        "questions": [{
+                            "id": "working-status-question",
+                            "title": "",
+                            "description": "What is the working status of this person?",
```

Comment:
This is the title not the description. It make the summary show incorrectly otherwise